src/features/MovieList/index.js

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from "react";
+import {
+  fetchPopularMovies,
+  selectPopularMoviesList,
+  selectPopularMoviesStatus,
+} from "./popularMoviesSlice";
+import { useDispatch, useSelector } from "react-redux";
+
+export const MovieList = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchPopularMovies());
+  }, []);
```

Comment:
czy tutaj zamiast [ ] nie powinno być [dispatch] ?

Reply:
z tego co pamiętam z lekcji, można dodać dispatch, ale nie jest to chyba konieczne i można to też tak zostawić, aczkolwiek z dispatchem faktycznie może wyglądać to lepiej :)